.changeset/heavy-seas-shave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/graphql-client': minor
+---
+
+Make fetch's keepalive configurable when making requests
```

Comment:
Let's add the example from the readme here as well!

Reply:
Done!